.github/workflows/package.yaml

```diff
@@ -0,0 +1,36 @@
+name: packaging
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  macos:
+
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8]
+
+    env:
+      PYTHON_VERSION: ${{ matrix.python-version }}
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: goanpeca/setup-miniconda@v1
```

Comment:
I think there is a miniforge one too

Reply:
CC @goanpeca

Reply:
>I think there is a miniforge one too

Not yet 🙃 but soon.

Also @xmnlab, please use

`conda-incubator/setup-miniconda@v1`

Reply:
thanks @scopatz and @goanpeca !